CONTRIBUTING.md

```diff
@@ -1,17 +1,44 @@
 # Contributing guidelines
 
-## Project structure
+To contribute, please follow this process:
 
-Moved to wiki: https://github.com/hyperledger/iroha/wiki/Project-structure
+## 1. Sign CLA agreement
+Please sign it via GitHub credentials: 
+[Developer Certificate of Origin Version 1.1](https://cla-assistant.io/hyperledger/iroha)
 
-## Developers guide
+## 2. Read Iroha working agreement
 
-Moved to wiki: https://github.com/hyperledger/iroha/wiki/Developers-guide
+[Iroha working agreement](https://github.com/hyperledger/iroha/wiki/Iroha-working-agreement)
 
-## Versioning policy
+## 3. Talk to us
+Before writing any code, please discuss your intention with us, so that we can help you with your solution. It can also help to reduce situations with duplicated effort.
+ 
+Do anything that is more convenient to you:
+* Join [Hyperledger RocketChat](https://chat.hyperledger.org/) #iroha channel
+* Open new issue in GitHub or discuss existing issue
+* Join telegram [Iroha chat](https://t.me/joinchat/AgzrTUCZ6efra09wz35oww) 
+* Use mailing list to discuss hyperledger-iroha@lists.hyperledger.org
+* Communicate in [Gitter chat](https://gitter.im/hyperledger-iroha/Lobby) with our development community 
 
-Releases are numbered according to the [Hyperledger standard release taxonomy](https://docs.google.com/document/d/1u9pt-bXeOXefYBB1uYE6M-D6CtmkC1lGCjmicSlZgVA/edit).
+## 4. Check codestyle
 
-## Branch policy
+Take a look briefly at [CppCoreGuidelines](https://github.com/isocpp/CppCoreGuidelines) and use [linter](https://github.com/hyperledger/iroha/blob/master/.clang-format) to check your code before creating pull request.
```

Comment:
clang-format is not a linter.
write `clang-format`.

... to check your code style...